src/store.ts

```diff
@@ -22,15 +23,17 @@ const IAB_VENDOR_LIST_PROD_URL =
     'https://www.theguardian.com/commercial/cmp/vendorlist.json';
 const IAB_VENDOR_LIST_NOT_PROD_URL =
     'https://code.dev-theguardian.com/commercial/cmp/vendorlist.json';
+
+let initialised = false;
+let source = DEFAULT_SOURCE;
+let variant: string | null = null;
```

Comment:
Question, should the default be null? If a user is not under test can we not include the `variant` property in the data we post to the consent logs or do we just set it to an empty string: `variant: ''`? @janua should be able to answer the prefered option?

Reply:
I set it to null as opposed to undefined so it's just as easy to check against and way easier to reset when we're unit testing the module. But what we're sending to the logs when this value is set to null is `{ variant: "" }` (check line 161 of this file).
I was working under the impression that we always need to pass a variant field but not sure if it was something @janua told us or something I assumed. Would be great if he could confirm.

Reply:
@GHaberis @ripecosta The consent logs side is either expecting a non-empty string value or the key completely missing. Null is unfortunately not accepted right now.

Reply:
👍 Will make the changes.